Sources/AblyAssetTrackingPublisher/Services/DefaultLocationService.swift

```diff
@@ -12,19 +12,46 @@ class DefaultLocationService: LocationService {
 
     init(mapboxConfiguration: MapboxConfiguration,
          historyLocation: [CLLocation]?,
-         logHandler: LogHandler?) {
-        
+         logHandler: LogHandler?,
+         vehicleProfile: VehicleProfile) {
+
         let directions = Directions(credentials: mapboxConfiguration.getCredentials())
-        NavigationSettings.shared.initialize(directions: directions, tileStoreConfiguration: .default)
-        
+
+
+        if vehicleProfile == .Bicycle {
+            NavigationSettings.shared.initialize(directions: directions, // or .shared
+                                                 tileStoreConfiguration: .default, // or .default
+                                                 navigatorPredictionInterval: 0,
+                                                 statusPollingConfig: .init(unconditionalPollingPatience: .greatestFiniteMagnitude ,
+                                                                            unconditionalPollingInterval: nil))
+            let cyclingConfig = [
+                        "cache": [
+                            "enableAssetsTrackingMode": true
+                        ],
+                        "navigation": [
+                            "routeLineFallbackPolicy": [
+                                "policy": 1
+                            ]
+                        ]
+                    ]
+            UserDefaults.standard.set(cyclingConfig, forKey: MapboxCoreNavigation.customConfigKey)
+        } else {
+            NavigationSettings.shared.initialize(directions: directions, tileStoreConfiguration: .default)
+        }
+
         if let historyLocation = historyLocation {
             replayLocationManager = ReplayLocationManager(locations: historyLocation)
         } else {
             replayLocationManager = nil
         }
         self.logHandler = logHandler
+        //set location manager with profile identifier only if .Bicycle is provided by clients
+        if vehicleProfile == .Bicycle {
+            self.locationManager = PassiveLocationManager(systemLocationManager: replayLocationManager ,datasetProfileIdentifier: .cycling)
```

Comment:
I don't know Swift so this question might be stupid but I just want to double-check: in the integration guide they are using the below code 
```
let manager = PassiveLocationManager(
            datasetProfileIdentifier: ProfileIdentifier.cycling
        )
```
and there they're using `ProfileIdentifier.cycling`, however, you have just `.cycling` 🤔 Is it the same thing or something different? 🤔

Reply:
It's exactly the same